src/main/scala/com/monsanto/arch/cloudformation/model/resource/AutoScaling.scala

```diff
@@ -12,8 +12,8 @@ case class `AWS::AutoScaling::AutoScalingGroup`(
     name:                    String,
     AvailabilityZones:       TokenSeq[String],
     LaunchConfigurationName: Token[ResourceRef[`AWS::AutoScaling::LaunchConfiguration`]],
-    MinSize:                 StringBackedInt,
-    MaxSize:                 StringBackedInt,
+    MinSize:                 Token[Int],
```

Comment:
Does this end up getting serialized properly into a string or do we need to add more code to support a `Token[StringBackedInt]`?

Reply:
Hi TJ,

We currently use this modified code in a template which is running in production:

Code:
![worksinproduction1](https://user-images.githubusercontent.com/13013300/51230534-980c2b80-1960-11e9-9e78-5a12bc0132e7.png)

Generated JSON:
![worksinproduction2](https://user-images.githubusercontent.com/13013300/51231209-ff2adf80-1962-11e9-8991-e53d087ddc98.png)

Although the [cloudformation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html) mentions these fields are expected to be Strings, it seems to work just as well with Number/Integer type (which makes more sense to me).